### PR TITLE
[IMP] Italian REA register

### PR DESCRIPTION
**Pull request by archetipo**

### CODE REVIEW

**Comment by archetipo** [comment]
:+1:

**Comment by archetipo** [comment]
:+1:

**Comment by tafaRU** [comment]
:+1:  once travis is green.

File: l10n_it_rea/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'REA Register',
+    'version': '0.1',
+    'category': 'Localisation/Italy',
+    'summary': 'Manage fields for  Economic Administrative catalogue',
+    'author': 'Agile Business Group',
```
Comment (eLBati) [inline l10n_it_rea/__openerp__.py]: Please add 'Odoo Community Association (OCA)'

Comment (archetipo) [inline l10n_it_rea/__openerp__.py]: fixed

**Comment by eLBati** [comment]
:+1: thanks

File: l10n_it_rea/res_partner.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+    _columns = {
+        'rea_office': fields.many2one(
+            'res.province', string='Office Province'),
+        'rea_code': fields.char('REA Code', size=20),
+        'rea_capital': fields.float('Capital'),
+        'rea_member_type': fields.selection(
+            [
+                ('unique', 'Unique Member'),
+                ('many', 'Multiple Members'),
```
Comment (eLBati) [inline l10n_it_rea/res_partner.py]: @archetipo these values should be SU, SM

File: l10n_it_rea/res_partner.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+class resPartner(orm.Model):
+    _inherit = 'res.partner'
+
+    _columns = {
+        'rea_office': fields.many2one(
+            'res.province', string='Office Province'),
+        'rea_code': fields.char('REA Code', size=20),
+        'rea_capital': fields.float('Capital'),
+        'rea_member_type': fields.selection(
```
Comment (eLBati) [inline l10n_it_rea/res_partner.py]: fixed with 09a56a7b5af9e1abb73869b6dca00f4edf972cbb

**Pull request merged**